oci/layer/generate.go

```diff
@@ -249,9 +249,9 @@ func GenerateInsertLayer(root string, target string, opaque bool, opt *RepackOpt
 					}
 					return ret
 				} else {
-					// skip this since previous layers dont have this file/dir
-					log.Debugf("skipping whiteout since %s not present in any previous layers", pathInTar)
-					return nil
+					// overlayfs adds opaque xattr for newly created dirs,
```

Comment:
does this path ever get hit for files and not just directories?

the old comment said "file/dir" but here we sound like we're assuming it's only dirs. Are there cases where we might have been missing adding a file to the tar also?

Reply:
Have seen this only for dirs.